### PR TITLE
Adds a dynamic config to cap the maximum threat level, max_threat_level

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

See title.

## Why It's Good For The Game

Config requested by the downstream.

## Changelog

:cl:
server: Dynamic now has an extra configuration value, max_threat_level. Defaults to 100 threat.
/:cl:

### CODE REVIEW

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Update dynamic/README.md to include it with the rest of the documentation.

**Comment by LemonInTheDark** [comment]
Does that var max out at 100?

**Comment by Iamgoofball** [comment]
I asked what to default it to and Mothblocks said 100.

**Review by Cyberboss — COMMENTED** [review]
Do we have a sample dynamic.json somewhere? If so, add this to it please

**Comment by Mothblocks** [comment]
@Cyberboss We don't need it there, there's a large amount of configuration options that aren't (considering everything that is a var is configurable). Good documentation is sound enough.

File: code/game/gamemodes/dynamic/readme.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ The "Dynamic" key has the following configurable values:
 - `random_event_hijack_minimum` - The minimum amount of time for antag random events to be hijacked. (See [Random Event Hijacking](#random-event-hijacking))
 - `random_event_hijack_maximum` - The maximum amount of time for antag random events to be hijacked. (See [Random Event Hijacking](#random-event-hijacking))
 - `hijacked_random_event_injection_chance` - The amount of injection chance to give to Dynamic when a random event is hijacked. (See [Random Event Hijacking](#random-event-hijacking))
-
+- `max_threat_level` - Sets the maximum amount of threat that can be rolled. Defaults to 100.
```
Comment (Mothblocks) [inline code/game/gamemodes/dynamic/readme.md:174]:
```suggestion
- `max_threat_level` - Sets the maximum amount of threat that can be rolled. Defaults to 100. You should only use this to *lower* the maximum threat, as raising it higher will not do anything.

```

**Pull request merged**